pyccel/parser/syntactic.py

```diff
@@ -176,10 +176,9 @@ def _treat_comment_line(self, line, stmt):
                     self._metavars[str(expr.name)] = expr.value
                     expr = EmptyNode()
             else:
-
-                raise errors.report(PYCCEL_INVALID_HEADER,
+                errors.report(PYCCEL_INVALID_HEADER,
                               symbol = stmt,
-                              severity='error')
+                              severity='fatal')
```

Comment:
Why did you increase the severity in this case?

Reply:
Good point.

From what I see [here](https://github.com/pyccel/pyccel/blob/26a14bfa9cae3e21b5175892b77f7bd1f7d334e2/pyccel/errors/errors.py#L296-L304) we now raise a `PyccelSyntaxError` which is a proper exception, while we were raising a `None` earlier.

Reply:
Yes that sounds right